### ISSUE TITLE
About import torch_geometric

### ISSUE THREAD

**Issue by cainiaoshidai** [rusty1s/pytorch_geometric#2482]
When I run `import torch_geometric`, I get the following error. I compiled and installed `torch_geometric` with source code. 

```
>>> import torch_geometric
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/gzm/anaconda3/envs/py3/lib/python3.7/site-packages/torch_geometric-1.7.0-py3.7.egg/torch_geometric/__init__.py", line 5, in <module>
    import torch_geometric.data
  File "/home/gzm/anaconda3/envs/py3/lib/python3.7/site-packages/torch_geometric-1.7.0-py3.7.egg/torch_geometric/data/__init__.py", line 6, in <module>
    from .dataloader import DataLoader, DataListLoader, DenseDataLoader
  File "/home/gzm/anaconda3/envs/py3/lib/python3.7/site-packages/torch_geometric-1.7.0-py3.7.egg/torch_geometric/data/dataloader.py", line 5, in <module>
    from torch._six import container_abcs, string_classes, int_classes
ImportError: cannot import name 'container_abcs' from 'torch._six' (/home/gzm/anaconda3/envs/py3/lib/python3.7/site-packages/torch/_six.py)

```
Thanks.

**Comment by cainiaoshidai** [comment]
I found the file of `_six.py`,  but these functions(`container_abcs, string_classes, int_classes`) are not found in this file.

**Comment by rusty1s** [comment]
Are you using PyTorch nightly? This seems to be related to https://github.com/rusty1s/pytorch_geometric/issues/1017.

**Comment by cainiaoshidai** [comment]
Thanks.
I successfully imported it after commenting out the relevant code.
Best.

**Comment by Borda** [comment]
seems we have a similar issue with geom 1.7 and PT 1.9
https://github.com/PyTorchLightning/lightning-tutorials/runs/2832212720

**Comment by rusty1s** [comment]
I will provide a new release in the following days supporting PyTorch 1.9.

**Comment by 0x18katyan** [comment]
> I will provide a new release in the following days supporting PyTorch 1.9.

Hey @rusty1s, sorry for the off-topic question. Will the edge_attr separation during train_test_split_edges be included in the release? If not, is there any way I can install nightly builds which support that functionality?

**Comment by rusty1s** [comment]
Yes, it will be included. You can install the "nightly" version of PyG via:
```
pip install git+https://github.com/rusty1s/pytorch_geometric.git
```

**Comment by Ademord** [comment]
came here because i am running the same problems with
ImportError: cannot import name 'container_abcs' from 'torch._six' (/home/ros/.local/lib/python3.8/site-packages/torch/_six.py)

**Comment by rusty1s** [comment]
Just released `torch-geometric==1.7.1` with PyTorch 1.9.0 support :)
